.github/workflows/Dockerfile.dev

```diff
@@ -37,5 +37,19 @@ RUN rustup target add x86_64-unknown-linux-musl && \
     rustup target add armv7-unknown-linux-musleabihf && \
     rustup target add x86_64-unknown-freebsd
 
+### Install Docker inside Docker
+RUN apt update;apt install -y docker.io && \
+    apt clean && \
+    rm -rf /var/lib/apt/lists/*
+##  Error starting systemctl daemon, lack of PID 1 cause of docker.
+##  But docker cli commands works cause of host running with -v /var/run/docker.sock:/var/run/docker.sock
+#   systemctl start docker && \
+
+### Add cross with Docker-in-Docker
+
+ENV CROSS_CONTAINER_IN_CONTAINER=true
+RUN cargo install cross && \
```

Comment:
By adding cross to  development image will this make freebsd binary generated? Or need someworkaround? As last conversation seems need to mount /var/run/docker.sock that like need different CI implementation.

Reply:
Yes, possible. It seems your suggestion with a separate yml and one plus workflow call will be the solution  here.

Maybe the best to close this PR and create a new one with that way. That can fix the strange file issue with the lldap rc.d script - where the latest commit not appears here, but exist in the PR source/fork repo.